templates/checkout/success.liquid

```diff
@@ -1,11 +1,21 @@
+<script>
+function sanitizeString(str){
```

Comment:
Seria interessante ser disponibilizado no tema todo afim de reutilizar como um helper.

Reply:
Verificar se a string é  null ou undefined, caso for, vai dar erro.